packages/web-react/src/components/Alert/useAlertStyleProps.tsx

```diff
@@ -6,19 +6,32 @@ import { SpiritAlertProps, AlertProps } from '../../types';
 export interface AlertStyles {
   /** className props */
   classProps: string;
+  /** Icon name to be passed to Icon */
+  iconName: string;
   /** props to be passed to the element */
   props: AlertProps;
 }
 
 export function useAlertStyleProps<T extends ElementType = 'div'>(props: SpiritAlertProps<T>): AlertStyles {
-  const { color, ...modifiedProps } = props;
+  const { color, iconName, isCentered, ...modifiedProps } = props;
 
   const alertClass = useClassNamePrefix('Alert');
   const alertColorClass = `${alertClass}--${color}`;
-  const classProps = classNames(alertClass, { [alertColorClass]: color });
+  const alertCenteredClass = `${alertClass}--center`;
+  const classProps = classNames(alertClass, {
+    [alertColorClass]: color,
+    [alertCenteredClass]: isCentered,
+  });
+
+  let iconNameValue = 'info';
```

Comment:
IMHO, placing this logic here is a bad idea. This hook handles only styles and should not know anything about icons. I think the better place will be the component itself or the new hook `useAlertIcon(color)`.

And also to the implementation: It will be better to use some kind of map, like key:value object where you can simply match color with icon name ;-)

Reply:
Updated, check it :)